src/Simulator/IncrementalRedstoneSimulator/IncrementalRedstoneSimulator.cpp

```diff
@@ -29,25 +29,48 @@
 
 
 
+const cRedstoneHandler * cIncrementalRedstoneSimulator::GetComponentHandler(BLOCKTYPE a_BlockType)
+{
+	struct sComponents:
+		public std::array<std::unique_ptr<cRedstoneHandler>, 256>
```

Comment:
No, I'm being an idiot, the current code uses null to signal block that can't interact with redstone at all. Ideally we'd move all the actual solid block handling code out of needing a dedicated handler because they are so sommon and have a different quick scan strategy, but that would be out of scope for this PR.